src/current/v23.2/cockroachdb-feature-availability.md

```diff
@@ -40,9 +40,9 @@ Command                                     | Description
 [`cockroach demo`]({% link {{ page.version.version }}/cockroach-demo.md %})     | Start a temporary, in-memory CockroachDB cluster, and open an interactive SQL shell to it.
 [`cockroach sqlfmt`]({% link {{ page.version.version }}/cockroach-sqlfmt.md %}) | Reformat SQL queries for enhanced clarity.
 
-### Custom Metrics Chart page for CockroachDB {{ site.data.products.serverless }} clusters
+### Custom Metrics Chart page for CockroachDB {{ site.data.products.cloud }} clusters
```

Comment:
Is it worth listing the metrics that are available to Dedicated, Serverless, or both? From @dhartunian's comment on the PR, it seems that we expect a similar list between Dedicated and Serverless? I wonder if it is worth saying "Serverless and Dedicated" clusters instead of conflating them all under "Cloud".